packages/clay-charts/src/svg/tiles.svg

```diff
@@ -1,55 +1,46 @@
 <svg xmlns="http://www.w3.org/2000/svg">
```

Comment:
Which would be the way to insert this in the DOM in an inert way? (not showing)

Reply:
@jbalsas for the moment we can use `<svg ... width="0" height="0">`  or `<svg ... style="display:none">` until we find the "idiomatic" way of doing it.